debian/po/templates.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: cloud-init\n"
 "Report-Msgid-Bugs-To: cloud-init@packages.debian.org\n"
-"POT-Creation-Date: 2021-03-22 14:08-0500\n"
+"POT-Creation-Date: 2021-04-20 10:14-0400\n"
```

Comment:
Why do we change this?

Reply:
I'm not really sure. I was following this commit.
e3e36b2870b70dc588c87ec492cf18a54f46395f